src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.scss

```diff
@@ -39,12 +39,18 @@
     z-index: 1;
 }
 
+.bcs-AnnotationActivity-link {
+    &::before {
+        margin-right: 8px;
+        margin-left: 8px;
+        color: $bdl-gray-50;
+        content: '\2022';
+    }
+}
+
 .bcs-AnnotationActivity-timestamp {
+    display: flex;
```

Comment:
Were we able to validate this works properly in IE11? It has some weird quirks around flexbox.

Reply:
Yup just confirmed it works as expected